bin/zopen-install

```diff
@@ -379,6 +379,11 @@ handlePackageInstall()
     printVerbose "Checking cache for already downloaded package [file name comparison]."
     location="zopen package cache"
   fi
+
+  # Check partition size before download package
+  megabytes=$(echo "scale=2; ${size} / (1024 * 1024)" | bc)
```

Comment:
Size is expandedsize (after extraction) from the json, but I think for this check, you'll need to get the pax size as in here: https://raw.githubusercontent.com/ZOSOpenTools/meta/main/docs/api/zopen_releases_latest.json